elitzur-core/src/main/scala/com/spotify/elitzur/validators/Validator.scala

```diff
@@ -16,9 +16,14 @@
  */
 package com.spotify.elitzur.validators
 
-import java.lang.{StringBuilder => JStringBuilder}
+import com.spotify.elitzur.validators.Validator.validateField
 
-import com.spotify.elitzur.{DataInvalidException, IllegalValidationException, MetricsReporter}
+import java.lang.{StringBuilder => JStringBuilder}
+import com.spotify.elitzur.{
```

Comment:
Sorry, running scalafmt on this made it seem like a much larger change. The material changes are in `SeqLikeValidator` (line 184),  `validationLoop` (line 327), and `validateField` (line 424)